tests/optimizers/test_optimizers.py

```diff
@@ -128,3 +129,13 @@ def test_report_failure(optimizer: Optimizer):
     assert isinstance(report.exception, ValueError)
     assert isinstance(report.traceback, str)
     assert report.metric_values == tuple(metric.worst for metric in optimizer.metrics)
+
+
+@parametrize_with_cases("optimizer", cases=".", prefix="opt_")
+def test_batched_ask_generates_unique_configs(optimizer: Optimizer):
+    """Test that batched ask generates unique configs."""
+    # NOTE: This was tested with up to 100, at least from SMAC and Optuna.
+    # It was quite slow for smac so I've reduced it to 10.
+    # This is not a hard requirement of optimizers (maybe it should be?)
+    batch = optimizer.ask(10)
+    assert all_unique(batch)
```

Comment:
This does not test that it also returns 10 trials, right?

Reply:
Ohh good point, should definitely include that!